src/ICeWall/doc/place_cell.md

```diff
@@ -0,0 +1,29 @@
+## place_cell
+### Synopsis
+```
+  % place_cell [-name <name>] \
+                     [-type <type>] \
```

Comment:
Please add an edge option so the correct cells can be chosen from the pad_library.

Reply:
If you want -edge as an option, this implies we're trying to add a cell into the padring, in whihc case I believe we should be using add_pad instead.

Reply:
Aren't name and type (cell? master? macro?) required args? They probably don't want the brackets around them to indicate they are optional (by my syntax rules anyway).

Reply:
Updated documentation - -inst_name <inst_name>is required,

If <inst_name> exists in the design and -cell is optional, if specified, then check that the specified cell type is being used
If <inst_name> does not exist in the design then -cell is required and an instance of <library_cell> is added to the design